tests/EmailTest.php

```diff
@@ -1,56 +1,107 @@
 <?php
-namespace emailApp;
-include dirname(__FILE__)."/../vendor/autoload.php";
+
+namespace EmailApp;
 
 class EmailTest extends \PHPUnit_Framework_TestCase {
-	
 	/**
+	 *This test validates email-id according to Gmail standards.
```

Comment:
Comments for tests are unnecessary. Test case function names are sufficient to explain what test it.